packages/sanity/src/ui/tab/Tab.tsx

```diff
@@ -0,0 +1,50 @@
+/* eslint-disable no-restricted-imports */
+import {Tab as UITab, TabProps as UITabProps} from '@sanity/ui'
+import React, {forwardRef} from 'react'
+
+/**
+ * @internal
+ *
+ * Padding and font sizes are fixed in Studio UI <Tab> components.
+ */
+export type TabProps = Pick<
+  UITabProps,
+  'aria-controls' | 'focused' | 'icon' | 'id' | 'label' | 'selected' | 'tone'
+> & {
+  size?: 'default' | 'small'
+}
+
+const SMALL_TAB_PROPS = {
+  padding: 2,
+}
```

Comment:
Changed screaming snake case for constants outside the component definition, which is a pattern we use throughout (and helps communicate intent). Also removed `muted: true` since this was shared across both.